packages/ods/src/components/link/src/components/ods-link/ods-link.scss

```diff
@@ -0,0 +1,37 @@
+@import '../../../../../style/link';
+
+.ods-link {
+  &__link {
+    @include ods-link;
+
+    &-text {
```

Comment:
in BEM you can only have either `--` or `__`
in your case, it as a new Element so use `&__text`